ui/app/templates/components/console/log-json.hbs

```diff
@@ -3,7 +3,7 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="console-ui-output has-text-grey">
+<div class="console-ui-output">
```

Comment:
great to see these helpers go away! for a component like this, specific styling makes more sense to me!